s3/src/main/java/ch/cyberduck/core/sts/STSCredentialsConfigurator.java

```diff
@@ -248,7 +248,7 @@ else if(!profiles.containsKey(basicProfile.getRoleSourceProfile())) {
                     log.debug(String.format("Configure credentials from basic profile %s", basicProfile.getProfileName()));
                 }
                 final Map<String, String> profileProperties = basicProfile.getProperties();
-                if(profileProperties.containsKey("sso_start_url") || profileProperties.containsKey("sso_session")) {
+                if(profileProperties.containsKey("sso_start_url")) {
```

Comment:
What is the reasoning for this change which seems unrelated. This applies to connections using STS  tokens obtained from cache of AWS CLI.